chapter_appendix-tools-for-deep-learning/jupyter.md

```diff
@@ -115,12 +115,25 @@ The above is the address of the remote server `myserver`. Then we can use http:/
 
 We can use the `ExecuteTime` plugin to time the execution of each code cell in a Jupyter Notebook. Use the following commands to install the plugin:
 
-```
+```sh
 pip install jupyter_contrib_nbextensions
 jupyter contrib nbextension install --user
 jupyter nbextension enable execute_time/ExecuteTime
 ```
 
+### Hinter(without using `tab`)
```

Comment:
space between Hinter and (

Reply:
@astonzhang i think we should move timer and this new section into the contribution guideline, instead of having it here.

Reply:
OK. He already has thought to revise. https://github.com/d2l-ai/d2l-en/pull/1101.

Reply:
Thanks. timer is useful because this book often compares executing speed. However, we do intend to keep nbextension minimal so I'd also suggest to not to include other plugin discussions in the main text.